### PR TITLE
P5: implementation-of-the-bitcoin-system

### LINKED ISSUE #9: 05-BTC-Implementation

### PR DESCRIPTION
**Pull request by hunknownz**
closes #9

### CODE REVIEW

File: BTC/implementation-of-the-bitcoin-system.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+# L5 Implementation of the Bitcoin System
```
Comment (wfnuser) [inline BTC/implementation-of-the-bitcoin-system.md:1]: please add a link in README.md

**Review by wfnuser — CHANGES_REQUESTED** [review]

**Comment by hunknownz** [comment]
closes #9

File: BTC/implementation-of-the-bitcoin-system.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+- Incentives in the Bitcoin System:
+	- Miners receive block rewards and transaction fees as incentives.
+	- As block rewards decrease, transaction fees will become the primary incentive for miners.
+	- Block Reward Halving:
+	- The block reward is halved every 210,000 blocks.
+	- This halving occurs approximately every 4 years, calculated as $$\frac{210000 \times 10  \text{ minutes}}{60  \text{ minutes} \times 24 \times 365} \approx 4 \text{ years}.$$
+
+## Bitcoin block information
+
+Bitcoin Block (548,402)
```
Comment (wfnuser) [inline BTC/implementation-of-the-bitcoin-system.md:49]: Is it a height? I think you should mark it as an example?

**Review by wfnuser — CHANGES_REQUESTED** [review]

File: BTC/implementation-of-the-bitcoin-system.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
+# L5 Implementation of the Bitcoin System
```
Comment (wfnuser) [inline BTC/implementation-of-the-bitcoin-system.md:1]: Final Decision: The title should be like P5 not L5.

Comment (hunknownz) [reply BTC/implementation-of-the-bitcoin-system.md:1]: done.

File: BTC/implementation-of-the-bitcoin-system.md (hunk 2 above)
Comment (hunknownz) [reply BTC/implementation-of-the-bitcoin-system.md:49]: done

**Review by wfnuser — APPROVED** [review]

**Pull request merged**